src/apps/ociochecklut/CMakeLists.txt

```diff
@@ -8,6 +8,14 @@ else()
     set(OCIO_GL_LIB oglapphelpers)
 endif()
 
+if(${OCIO_USE_HEADLESS})
+    if(${OCIO_EGL_HEADLESS})
+        add_compile_definitions(OCIO_HEADLESS_ENABLED)
+    else()
+        message(WARNING "EGL component missing. Can't render headlessly for ociochecklut.")
+    endif()
+endif()
+
```

Comment:
The code must fallback smoothly to GLX if EGL is not found so the test is only:
```
if(OCIO_EGL_HEADLESS)
   add_compile_definitions(OCIO_HEADLESS_ENABLED) 
endif()
```

The message must be in the root CMakeLists.txt i.e. to appear only once at the beginning.